### PR TITLE
fix(modal): center evolution sprites inside modal for all cases

### LINKED ISSUE #147: Adjust pokemon evolution display
**Is your feature request related to a problem? Please describe.**
For pokemon with 2 evolutions, the modal shows the spacing of the sprites very far off. Single pokemon sprite should be centered.
This should be adjusted for consistency.
image
Screen Shot 2021-08-17 at 9 22 49 AM

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Fix sprite evolution positioning, to where sprites are centered for all cases of evolution sprite numbers.

Closes #147

### CODE REVIEW

File: src/styles/components/_modal.scss [hunk 1]
```diff
@@ -137,9 +137,9 @@ $modal-padding: 25px;
 .spriteContainer {
   display: flex;
   margin: 25px;
-  align-content: center;
   align-items: center;
-  justify-content: space-between;
+  justify-content: safe center;
```
Comment (BlackCubes) [inline src/styles/components/_modal.scss:141]: Interesting. Never seen `safe`

**Review by BlackCubes — APPROVED** [review]
Good job on the fix!

File: src/styles/components/_modal.scss (hunk 1 above)
Comment (kmausolf) [reply src/styles/components/_modal.scss:141]: Very interesting!  https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

**Review by connielion — APPROVED** [review]
Interesting...

File: src/styles/components/_modal.scss (hunk 1 above)
Comment (dannysanchez559) [reply src/styles/components/_modal.scss:141]: Yep, the only css solution to fix our problem, where the eevee sprites were longer than the flex container!

**Pull request merged**